### PR TITLE
feat: ssh tunnel shortcuts

### PR DESCRIPTION
**Pull request by pgierz**

### CODE REVIEW

File: src/pymorize/ssh_tunnel.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@click.option("--username", required=True, help="Username for SSH connections")
+def ssh_tunnel_cli(local_port, remote_port, gateway, compute_node, username):
+    """
+    Create an SSH tunnel to access a Dask dashboard on a remote compute node.
+
+    This script sets up an SSH tunnel to access a Dask dashboard running
+    on a remote compute node, routed through a gateway server.
+    """
+    ssh_command = (
+        f"ssh -nNT -L {local_port}:{compute_node}:{remote_port} {username}@{gateway}"
```
Comment (siligam) [inline src/pymorize/ssh_tunnel.py:20]: ###  how about using ControlMaster and ControlPath while creating the connection
"ssh -nNT -M -S tmp_tunnel -L {local_port}:{compute_node}:{remote_port} {username}@{gateway}"

File: src/pymorize/ssh_tunnel.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    click.echo(f"Dashboard will be accessible at http://localhost:{local_port}/status")
+    click.echo("Press Ctrl+C to close the tunnel")
+
+    try:
+        # Run the SSH command
+        subprocess.run(ssh_command, shell=True, check=True)
+    except subprocess.CalledProcessError as e:
+        click.echo(f"An error occurred while creating the SSH tunnel: {e}")
+    except KeyboardInterrupt:
+        click.echo("SSH tunnel closed.")
```
Comment (siligam) [inline src/pymorize/ssh_tunnel.py:36]: ### now using the ControlPath to close the connection
subprocess.run(f"ssh -S ./tmp_tunnel -O exit {gateway}", shell=True, check=True)

Comment (pgierz) [reply src/pymorize/ssh_tunnel.py:36]: Closing this comment, adding @siligam's suggestion above.

Comment (pgierz) [inline src/pymorize/ssh_tunnel.py:36]:
```suggestion
        subprocess.run(f"ssh -S ./tmp_tunnel -O exit {gateway}", shell=True, check=True)
        click.echo("SSH tunnel closed.")
```

File: src/pymorize/ssh_tunnel.py (hunk 1 above)
Comment (siligam) [reply src/pymorize/ssh_tunnel.py]: I guess it will close the ports gracefully. I am not 100% sure but could give a try.

Comment (pgierz) [reply src/pymorize/ssh_tunnel.py]: The ports close regularly after a Control + C, so I don't think we need to complicate the SSH command too much by using this.

**Comment by pgierz** [comment]
This pull request updates the `ssh_tunnel.py` script to support forwarding ports for both Dask and Prefect dashboards. The changes include adding new options for specifying local and remote ports for Prefect, modifying the gateway server hostname, and updating the SSH command and output messages accordingly.

### Enhancements to SSH Tunnel Script:

* `src/pymorize/ssh_tunnel.py`: 
  - Added options for specifying local and remote ports for Prefect (`--local-prefect-port` and `--remote-prefect-port`).
  - Changed the default gateway server hostname from `levante2.dkrz.de` to `levante1.dkrz.de`.
  - Updated the SSH command to include port forwarding for Prefect in addition to Dask.
  - Modified the output messages to reflect the new port forwarding setup and provide URLs for accessing both Dask and Prefect dashboards.

**Pull request merged**